prerendering.bs

```diff
@@ -658,6 +673,41 @@ The <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> gette
 
 1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
 
+<h3 id="client hint cache">Interaction with Client Hint Cache</h3>
+
+We need to ensure that the [=Accept-CH cache=], which is owned by user agent as a global storage, is not modified while prerendering, but is properly updated after activation. The following modifications ensure this:
+
+Each [=prerendering browsing context=] maintains its <dfn for="prerendering browsing context">prerender-scoped Accept-CH cache</dfn>, to persist the origin's opt-in to receive the indicated header fields.
+
+The [=prerendering browsing context/prerender-scoped Accept-CH cache=] is an [=ordered map=] of [=origins=] to [=client hints sets=].
+
+<div algorithm="add a new prerender Accept-CH cache entry">
```

Comment:
nit: if there's only one dfn in the div, the name of the algorithm is automatically inferred, so you can just write `<div algorithm>`

Reply:
Thanks! :D got it!